postgres/postgres_test.go

```diff
@@ -54,6 +54,22 @@ func TestGetInstance(t *testing.T) {
 	})
 }
 
+func TestGetInstances(t *testing.T) {
+	t.Parallel()
+	Convey("When a request for all, a list of instances are returned", t, func() {
+		jsonContent := "{ \"state\":\"Created\"}"
```

Comment:
Might not be something we need to fix right now, but job states should all be lower case. As long as it's consistent it's not a problem for now, but definitely one to put on the list to sort soon.